src/pytype.jl

```diff
@@ -363,13 +363,46 @@ function pyjlwrap_hash32(o::PyPtr)
     return h == reinterpret(UInt32, Int32(-1)) ? pysalt32 : h::UInt32
 end
 
+# just string(Docs.doc(f)) doesn't work on VERSION < v"0.5"
+function docstring(f::Function)
+    buf = IOBuffer()
+    show(buf, "text/plain", Docs.doc(f))
+    return String(take!(buf))
+end
+
+# this function emulates standard attributes of Python functions,
+# where possible.
+function pyjlwrap_getattr(self_::PyPtr, attr__::PyPtr)
+    attr_ = PyObject(attr__) # don't need pyincref because of finally clause below
+    try
+        f = unsafe_pyjlwrap_to_objref(self_)
+        attr = convert(String, attr_)
+        if attr in ("__name__","func_name")
+            return pystealref!(PyObject(string(f)))
+        elseif attr in ("__doc__", "func_doc")
+            return pystealref!(PyObject(docstring(f)))
+        elseif attr in ("__module__","__defaults__","func_defaults","__closure__","func_closure")
+            return pystealref!(PyObject(nothing))
+        else
```

Comment:
Done. I retained this code path for things that start with `__`, because it seems like python needs that to work correctly. If you'd prefer, we can probably find the complete list of such things, but that seemed easier.